exchanges/exchange.go

```diff
@@ -1333,6 +1336,57 @@ func (e *Endpoints) GetURLMap() map[string]string {
 	return urlMap
 }
 
+// GetCachedOpenInterest returns open interest data if the exchange
+// supports open interest in ticker data
+func (b *Base) GetCachedOpenInterest(_ context.Context, k ...key.PairAsset) ([]futures.OpenInterest, error) {
```

Comment:
Suggestion: There might be an issue when you are not actively managing tickers and using this package as a wrapper only. It might be better to have this as a higher order function call IBotExchange instead of  contained in individual wrapper implementation.

Reply:
Its why it has such a polite fallback in `GetOpenInterest` IBotExchange implementations. If its not being managed, it just returns and does the API calls.
Where are you suggesting it be placed and called from? I appreciate its current spot to save on API calls by just returning highly likely rapidly updated tickers than making a new round-trip

Reply:
For some reason I missed that polite fall back. Will resolve this nonsense request.